internal/worker/caasapplicationprovisioner/ops.go

```diff
@@ -694,9 +695,13 @@ func ensureScale(appName string, app caas.Application, appLife life.Value,
 		ps = &params.CAASApplicationProvisioningState{}
 	}
 
+	info, err := facade.ProvisioningInfo(appName)
```

Comment:
We should only fetch what's needed to do the work. So we would need to implement a new facade method just to get the attachment info